apps/dashboard/app/controllers/system_status_controller.rb

```diff
@@ -0,0 +1,7 @@
+class SystemStatusController < ApplicationController
+  def index
+    @job_clusters = OodAppkit.clusters
+                             .select(&:job_allow?)
+                             .reject { |c| c.metadata.hidden }
```

Comment:
Looks like you pulled this from here? Seems like we shouldn't reimplement it or in fact even need it here - the view can directly use `Configuration.job_clusters.each`.


https://github.com/OSC/ondemand/blob/21216959410fcfd347739e8195c68f746305ff11/apps/dashboard/config/configuration_singleton.rb#L115-L121